### PR TITLE
Fix: UnicodeDecodeError occurs when running gitman update/install in …

### PR DESCRIPTION
**Pull request by sch0125**
…Windows

### CODE REVIEW

**Comment by sch0125** [comment]
The default codepage for Windows console is CP1252 in which 0x9d is undefined.

From [python documentation ](https://docs.python.org/3/library/subprocess.html):

> class subprocess.Popen(args, bufsize=- 1, executable=None, stdin=None, stdout=None, stderr=None, preexec_fn=None, close_fds=True, shell=False, cwd=None, env=None, universal_newlines=None, startupinfo=None, creationflags=0, restore_signals=True, start_new_session=False, pass_fds=(), *, group=None, extra_groups=None, user=None, umask=- 1, encoding=None, errors=None, text=None, pipesize=- 1)

>> If encoding or errors are specified, or text is true, file objects for stdin, stdout and stderr are opened in text mode using the specified encoding and errors or the [io.TextIOWrapper](https://docs.python.org/3/library/io.html#io.TextIOWrapper) default. The universal_newlines argument is equivalent to text and is provided for backwards compatibility. By default, file objects are opened in binary mode.

>> Changed in version 3.6: Added encoding and errors parameters

>> Changed in version 3.3: When universal_newlines is True, the class uses the encoding [locale.getpreferredencoding(False)](https://docs.python.org/3/library/locale.html#locale.getpreferredencoding) instead of locale.getpreferredencoding(). See the [io.TextIOWrapper](https://docs.python.org/3/library/io.html#io.TextIOWrapper) class for more information on this change.

Add encoding=utf-8 to Popen() to fix #271

File: gitman/shell.py [hunk 1]
```diff
@@ -32,6 +32,7 @@ def call(name, *args, _show=True, _stream=True, _shell=False, _ignore=False):
 
     command = subprocess.Popen(  # pylint: disable=subprocess-run-check
         name if _shell else [name, *args],
+        encoding='utf-8',
```
Comment (Erich-McMillan) [inline gitman/shell.py:35]: Looks like CI enforces `"` formatting like so:
```suggestion
        encoding="utf-8",
```

**Review by Erich-McMillan — APPROVED** [review]

**Comment by Erich-McMillan** [comment]
@jacebrowning looks like Travis CI checks are failing for python3.7.1, I think pip installation is corrupted? https://app.travis-ci.com/github/jacebrowning/gitman/jobs/560809820

**Comment by jacebrowning** [comment]
@Erich-McMillan I cleared the build cache and now it seems to be working. I'll get this fix released.

**Review by jacebrowning — APPROVED** [review]

**Pull request merged**